core/pom.xml

```diff
@@ -100,7 +100,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.apache.directory.server</groupId>
```

Comment:
Without it, running tests creates the following exception:

    java.lang.NoClassDefFoundError: org/apache/directory/server/core/api/DirectoryService

Is there a different package I should be importing?